packages/cyberstorm/src/components/Tooltip.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { PropsWithChildren, ReactNode } from "react";
+import * as RadixTooltip from "@radix-ui/react-tooltip";
+import styles from "./componentStyles/Tooltip.module.css";
+
+export interface TooltipProps {
+  content?: ReactNode;
+  sideOffset?: number;
+  side?: string;
+  collisionPadding?: number;
+  sticky?: string;
+  avoidCollisions?: boolean;
+}
+
+// Cyberstorm Tooltip component
+
+export const Tooltip: React.FC<PropsWithChildren<TooltipProps>> = ({
+  content,
+  sideOffset = 5,
+  side = "right",
+  collisionPadding = 20,
+  sticky = "always",
+  avoidCollisions = true,
+  children,
+}) => {
+  return (
+    <RadixTooltip.Root>
+      <RadixTooltip.Trigger>{children}</RadixTooltip.Trigger>
+      <RadixTooltip.Portal>
+        <RadixTooltip.Content
+          className={`${styles.root}`}
```

Comment:
Why is string interpolation used here? You can just use the variable directly: `className={styles.root}`

Reply:
Fixed this :D